OJP/OJP_Common.xsd

```diff
@@ -283,11 +283,12 @@
 			</xs:element>
 		</xs:sequence>
 	</xs:group>
-	<xs:complexType name="SharingServiceStructure">
+	<xs:complexType name="AlternativeServiceStructure">
 		<xs:annotation>
 			<xs:documentation>Service that provides shared vehicles (The Transmodel MODE OF OPERATION is VEHICLE SHARING; other related Transmodel concepts: ALTERNATIVE MODE LEG.SINGLE JOURNEY.COMMON.VEHICLE SERVICE.TRANSPORT ORGANISATION).</xs:documentation>
 		</xs:annotation>
 		<xs:sequence>
+			<xs:element name="Mode" type="IndividualModeEnumeration"/>
```

Comment:
This doesn't allow for "schoolBus", "companyShuttle", etc. Below there is the `Restricted` element, so such private modes should be handled somewhere.

Reply:
This works, because 
ContinuousServiceStructure has a choice between
* AlternativeModeOfOperation or
* PersonalModeOfOperation

before there is the choice between:
* DatedJourneyGroup
* IndividualMode
* AlternativeService

In combination between
AlternativeService/Mode   and AlternativeModeOfOperation I think all these should be expressable. Or do you think something is missing?  @trurlurl

Reply:
I see that the use of a "companyShuttle" and other "private modes" can be expressed as a `AlternativeModeOfOperation" elsewhere. But then, in my view, one should remove all mentions of "private modes" and "PrivateMode" (e.g., in `PrivateServiceGroup` or in the annotation of `IncludeAllRestrictedLines`, see #349). Private modes wouldn't exist any more, only "private modes of operation". And perhaps that wouldn't be needed any more either - just speaking of of modes of operation that may be restricted somehow, as will be signalled in a `RestrictionNote`?

As long as one uses the term "private mode", I personally would expect to find it here in the element `Mode`. That was my problem.